core/app/models/spree/wallet/add_after_order_complete.rb

```diff
@@ -0,0 +1,25 @@
+class Spree::Wallet::AddAfterOrderComplete
```

Comment:
@tvdeyen @peterberkenbosch hm, I am not sure about this one.  Right now the logic for this is very dependent on this happening during order completion.  `Wallet::AddAfterOrderComplete` seems like it might be the best we can get right now.  What if we leave it like this for the moment and take another stab at improving things as part of the Wallet work?